aws/terraform/metaflow/example.tfvars

```diff
@@ -0,0 +1,21 @@
+env = "prod"
```

Comment:
Just a larger question - Would it make sense to reorganise the templates a little bit - 
`aws/terraform/README.md` - documentation for the overall template
`aws/terraform/infra.tfvars` - top level vars file
`aws/terraform/infra.tf` - top level terraform script

`aws/terraform/metaflow-service/README.md` - README for metaflow service
`aws/terraform/metaflow-service/infra.tfvars` - terraform vars for metaflow service
`aws/terraform/metaflow-service/infra.tf` - terraform script for metaflow service

`aws/terraform/compute/batch/README.md` - README for Batch
`aws/terraform/compute/batch/infra.tfvars` - terraform vars for Batch
`aws/terraform/compute/batch/infra.tf` - terraform script for Batch

`aws/terraform/datastore/README.md` - README for s3
`aws/terraform/datastore/infra.tfvars` - terraform vars for s3
`aws/terraform/datastore/infra.tf` - terraform script for s3

`aws/terraform/scheduling/README.md` - README for sfn
`aws/terraform/scheduling/infra.tfvars` - terraform vars for sfn
`aws/terraform/scheduling/infra.tf` - terraform script for sfn

...

Just an idea. Maybe we can address it later.

Reply:
Oh looks like the changes have been made below.

Reply:
looks like some of the files (like this one) are redundant?

Reply:
I think Kelly's comment here may help explain why these files seem redundant

https://github.com/Netflix/metaflow-tools/pull/26#issuecomment-840533001

Any thoughts post reading that?